10-kitten/Containerfile

```diff
@@ -1,27 +1,24 @@
-FROM docker.io/library/almalinux:10-kitten as repos
-FROM quay.io/centos-bootc/bootc-image-builder:latest as builder
+FROM quay.io/almalinuxorg/10-kitten-base:10-kitten as repos
+FROM quay.io/centos-bootc/centos-bootc:stream10 as builder
 
+RUN rm -rf /etc/yum.repos.d/*
 
-COPY --from=repos /etc/dnf/vars /etc/dnf/vars
+COPY --from=repos /etc/yum.repos.d/*.repo /etc/yum.repos.d/
 
-COPY 10-kitten /src
-COPY fedora-bootc /src/fedora-bootc
-WORKDIR /src
-
-COPY --from=repos /etc/yum.repos.d/*.repo /src
 COPY --from=repos /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux-10 /etc/pki/rpm-gpg
 
-RUN --mount=type=cache,target=/workdir \
-    --mount=type=bind,rw=true,src=.,dst=/buildcontext,bind-propagation=shared \
-    rpm-ostree compose image \
-        --image-config almalinux-bootc-config.json \
-        --cachedir=/workdir \
-        --format=ociarchive \
-        --initialize almalinux-bootc.yaml \
-        /buildcontext/out.ociarchive
+COPY 10-kitten/almalinux-10-kitten.yaml /usr/share/doc/bootc-base-imagectl/manifests/
+
+RUN /usr/libexec/bootc-base-imagectl build-rootfs --manifest=almalinux-10-kitten /target-rootfs
+
+###
+
+FROM scratch
+
+COPY --from=builder /target-rootfs/ /
 
-FROM oci-archive:./out.ociarchive
+LABEL containers.bootc 1
+LABEL ostree.bootable 1
 
-RUN --mount=type=bind,from=builder,src=.,target=/var/tmp \
-    --mount=type=bind,rw=true,src=.,dst=/buildcontext,bind-propagation=shared \
-    rm /buildcontext/out.ociarchive
+STOPSIGNAL SIGRTMIN+3
+CMD ["/sbin/init"]
```

Comment:
Total nitpick, but could you add a final newline to the files that are missing one?

Reply:
Sure

Reply:
Added

Reply:
GitHub appears to hide ending newlines in some cases, but they are present locally.